apps/adobe/win.py

```diff
@@ -0,0 +1,41 @@
+from talon import Context, actions
+
+# Context matching
+ctx = Context()
+ctx.matches = """
+os: windows
+app: adobe_acrobat_reader_dc
+"""
+
+
+# --- Implement actions ---
+@ctx.action_class('app')
+class AppActions:
+    # app.tabs
+    def tab_next(): actions.key('ctrl-tab')
+    def tab_previous(): actions.key('ctrl-shift-tab')
+
+
+@ctx.action_class('edit')
+class EditActions:
+    def zoom_in(): actions.key("ctrl-0")
```

Comment:
Shouldn't zoom out/in be `ctrl--`/`ctrl-+`?

Reply:
There seems to be conflicting information on the internet. At least for me it is ctrl-0/1 (german version). 
![grafik](https://user-images.githubusercontent.com/24251362/141747737-ebf39fe4-46b9-44c6-972a-5b5c945df7fd.png)

Reply:
Well, that's irritating. I'm honestly not sure how to handle this sort of thing. Maybe live with this for now, and submit an issue? Open to suggestions.

Reply:
Yeah that's frustrating.  Not sure what the right way to handle this one is.  Is ctrl-0/1 the common way to zoom in / out on German version of Windows?  If so I guess we could add a tag for German OS or something and make that global?

Reply:
Any other app uses `ctrl-+` and `ctrl--`. Only Acrobat is weird.

Reply:
But there are other shortcuts that differ per language. For example the "window next" one.

Reply:
Maybe a tag or maybe there's a (windows?) API that could be used to determine the current language. In any case I think this can be dealt with in the next PR.